lib/ui_components/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module UiComponents
-  VERSION = '2.2.1'
+  VERSION = '2.2.2'
```

Comment:
@tsujigiri shouldn't change anything about how to use it.
@RKushnir i will increase the minor version anyway - it's true that it's a fairly significant change.